02_configure_host.sh

```diff
@@ -21,11 +21,13 @@ early_deploy_validation
 manage_libvirtd() {
   case ${DISTRO} in
       centos9|rhel9)
-          for i in qemu network nodedev nwfilter secret storage interface; do
+          for i in qemu interface network nodedev nwfilter secret storage; do
```

Comment:
Just add proxy to this list?

Reply:
Yes, we could do that. Will update in the next patch.